packages/frontend/src/utils/account.test.js

```diff
@@ -0,0 +1,20 @@
+import { shortenAccountId } from './account';
+
+test('ID is shorten with correct formats', () => {
+    const sourceId = '4ee68bC02dd892aa1ad819101f3adf4cfafb90e3f73b827f4c2352b6efe363ek';
+
+    expect(shortenAccountId(sourceId)).toBe('4ee68bC0...efe363ek');
+    expect(shortenAccountId(sourceId, 7, 4)).toBe('4ee68bC...63ek');
+    expect(shortenAccountId(sourceId, 13, 1)).toBe('4ee68bC02dd89...k');
+    expect(shortenAccountId(sourceId, 30, 31)).toBe('4ee68bC02dd892aa1ad819101f3adf...afb90e3f73b827f4c2352b6efe363ek');
+});
+
+test('ID remains the same', () => {
```

Comment:
naming: "It should remain wrong accountId unchanged"

Reply:
Set it as: "It should not change account ID". Because "unchanged state" depends not only on the wrong account ID.